resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -0,0 +1,232 @@
+# =========================================
+# Regtools - Junctions Extract and Annotate
+# =========================================
+---
+
+- slug: regtools-junctions-annotate
+  name: Annotate novel splice junctions (regtools)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:3.7.0
+  data_name: "Novel Junctions ({{ star|sample_name|default('') }}{{ extracted|sample_name|default('') }})"
+  version: 0.0.1
+  type: data:bed:junctions
+  category: Splice Junctions
+  flow_collection: sample
+  persistence: CACHED
+  description:
+    Identify novel splice junctions by using regtools to annotate against a reference. The process
+    accepts Reference genome, Reference genome annotation (GTF), and input with reads information
+    (STAR aligment or reads aligned by any other aligner or junctions in BED12 format). If STAR aligner
+    data are defined as input, the process calculate BED12 file from STAR SJ.out.tab file, and annotate
+    all junctions with regtools annotate command. When reads are aligned by other aligner, junctions
+    are extracted with regtools junctions extract tool and then annotated with junction annotate command.
+    Third option allow user to provide directly BED12 file with junctions, which are then annotated.
+    Finnally, annotated novel junctions are filtered in separate output file. More information can
+    be found in the [regtools manual](https://regtools.readthedocs.io/en/latest/).
+  input:
+    - name: star_junctions
+      label: STAR alignment
+      type: data:alignment:bam:star
+      description: >
+        STAR alignment with SJ.out.tab output. Required if a .bed junctions file is not provided.
+      required: false
+    - name: aligned_reads
+      label: Aligned reads (BAM) file
```

Comment:
I favour third option. But Aligned reads (STAR) is not correct, because the input is `SJ.out.tab` file from STAR output. I have changed it to  `Splice junctions extracted by STAR aligner`.